src/ui/components/shared/WorkspaceSettingsModal/WorkspaceSettingsModal.tsx

```diff
@@ -130,7 +129,6 @@ const settings: Settings<
     settings?: any;
     isAdmin: boolean;
     workspaceId: string;
-    hideModal: PropsFromRedux["hideModal"];
```

Comment:
If I read this correctly... only `TeamMembers` were actually using `hideModal` so I've just moved it directly to it and cleaned this up from the parent components